src/app/api/og/route.tsx

```diff
@@ -144,6 +157,192 @@ const MeetingOGImage = async (cityId: string, meetingId: string) => {
     );
 };
 
+// Meeting Story OG Image (Vertical - 1080x1920 for Instagram Stories)
+const MeetingStoryOGImage = async (cityId: string, meetingId: string) => {
+    const data = await getMeetingDataForOG(cityId, meetingId);
+    if (!data) return null;
+
+    const meetingDate = new Date(data.dateTime);
+    const formattedDate = meetingDate.toLocaleDateString('el-GR', {
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+    });
+
+    // Sort subjects by hotness
+    const sortedSubjects = [...data.subjects].sort((a, b) => {
+        if (a.hot && !b.hot) return -1;
+        if (!a.hot && b.hot) return 1;
+        return 0;
+    });
+    const topSubjects = sortedSubjects.slice(0, 5);
+    const remainingCount = Math.max(0, data.subjects.length - 5);
```

Comment:
We’re already using similar logic for `MeetingOGImage`, but now that we have this helper https://github.com/schemalabz/opencouncil/blob/c4f20222963abeee0ef431714c3da38085891660/src/lib/utils.ts#L91 this would be a good opportunity to replace the custom sorting logic for meeting OG images with it. That way we stay consistent across the codebase and ensure the most important subjects are surfaced.